text/0034-telemetry-policies.md

```diff
@@ -186,6 +189,8 @@ The collected data is sent to [Segment](https://segment.com/). Segment is a plat
 | `per_batch` | `true` if `POST /indexes/:indexUid/documents/delete-batch` endpoint was used in this batch, otherwise `false` | false | `Documents Deleted` |
 | `per_filter`| `true` if `POST /indexes/:indexUid/documents/delete` endpoint was used in this batch, otherwise `false` | false | `Documents Fetched GET`, `Documents Fetched POST`, `Documents Deleted` |
 | `clear_all` | `true` if `DELETE /indexes/:indexUid/documents` endpoint was used in this batch, otherwise `false` | false | `Documents Deleted` |
+| vector_store | Whether the [vector store](./0193-experimental-features.md#vector-store) feature is enabled. | `true` | `Experimental features Updated` |
```

Comment:
@dureuill we just did a telemetry QA and it looks like `vector_store` and `score_details` are camel-cased in Amplitude. I think they should match the spec?